Schema/V1/metadatakatalog.xsd

```diff
@@ -666,6 +643,31 @@
     </xs:sequence>
   </xs:complexType>
 
+  <xs:complexType name="referanseTilKlassifikasjon">
```

Comment:
Er denne i bruk?

Reply:
Ja. Her: https://github.com/ks-no/fiks-arkiv-specification/blob/7f24d1b3ed69ac1a11b5c588a210c327832c4ac1/Schema/V1/no.ks.fiks.arkiv.v1.arkivering.arkivmelding.oppdater.xsd#L254

Reply:
Sorry, tenkte på referanseobjektet og ikke på angivelsen av klassifikasjon for å kunne endre denne på en sak. Da er den OK.